src/main/java/com/omvl/omvl/controller/MemberController.java

```diff
@@ -21,51 +28,66 @@ public MemberController(MemberService memberService) {
 	}
 
 	//회원가입 페이지로 이동
-	@GetMapping("/members/new")
+	@GetMapping("/new")
 	public String createForm() {
 		return "members/createMemberForm";
 	}
 
 	//회원가입 진행
-	@PostMapping("/members/new")
-	public String create(MemberForm form) {
-		Member member = new Member();
-
-		member.setId(form.getId());
-		member.setPassword(form.getPassword());
-		member.setName(form.getName());
-		member.setAge(form.getAge());
-		member.setGender(form.getGender());
+	@PostMapping("/new")
+	public String create(@ModelAttribute Member member) {
 
 		memberService.join(member);
 
 		return "redirect:/";
 	}
 
+	//아이디 중복 체크
+	@PostMapping("/checkId")
+	public ResponseEntity<Map<String, Boolean>> checkIdDuplication(@RequestParam("memberId") String memberId) {
+
+		boolean isDuplicated = memberService.validateDuplicateMember(memberId);
```

Comment:
`validateDuplicateMember` 네이밍은 검사를 하고 잘못 됐으면 Exception을 Throw 할 것 같은 이름이네요. 단지 중복 됐는지를 묻는거라면 `duplicated` 또는 `isDuplicated` 는 어떤가요? 개인적으로 Boolean 응답에는 is를 생략하고 단순 형용사 메소드 네이밍이 가장 적절하다고 생각해요. 그러면 사용하는 곳에서도 좀 더 잘 읽힐 것 같아요.
```suggestion
		boolean isDuplicated = memberService.duplicated(memberId);
```

Reply:
넵!!